File: poc-challenges/250727/src/test/java/com/tdd/domain/FibonacciTest.java
package com.tdd.domain;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class FibonacciTest {
	@Test
	void fibonacci_test() {
		assertEquals(0, Fibonacci.fib(0));
		assertEquals(1, Fibonacci.fib(1));
		assertEquals(1, Fibonacci.fib(2));
		assertEquals(2, Fibonacci.fib(3));
	}
}

File: poc-challenges/250916/src/test/kotlin/com/currenjin/tdd/FibonacciTest.kt
package com.currenjin.tdd

import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class FibonacciTest {
    @Test
    fun fibonacci() {
        assertEquals(0, Fibonacci.fib(0))
        assertEquals(1, Fibonacci.fib(1))
    }
}